### ISSUE TITLE
Super Mario Advance 4 - Super Mario Bros. 3 (USA) (Wii U Virtual Console)

### ISSUE THREAD

**Issue by fr500** [libretro/libretro-database#847]
Super Mario Advance 4 - Super Mario Bros. 3 (USA) (Wii U Virtual Console) matches with a hack (because the CRC is identical).

How do I make it match the actual WiiU VC one instead?
Remove the hacks db?

**Comment by RobLoach** [comment]
Yes, the Hacks DB was removed a while ago. Unfortunately hitting "Update Database" doesn't remove old ones for you. What is the CRC that you have?
